### PR TITLE
Always generate absolute url for logging

### PR DESCRIPTION
**Pull request by oleg-andreyev**
Always generate Absolute URL.

---

~~In some cases it's required to generate Absolute URL.~~
~~This PR is adding `use_absolute_url` configuration parameter, which affects `nelmio_js_logger_log` -result (`ABSOLUTE_URL` or `ABSOLUTE_PATH` (default))~~

### CODE REVIEW

**Comment by oleg-andreyev** [comment]
ping @Seldaek

File: TwigExtension.php [hunk 1]
```diff
@@ -228,4 +231,9 @@ public function getName()
     {
         return 'nelmio_js_logger';
     }
+
+    private function generateUrl()
+    {
+        return addslashes($this->router->generate('nelmio_js_logger_log', [], $this->useAbsoluteUrl ? UrlGeneratorInterface::ABSOLUTE_URL : UrlGeneratorInterface::ABSOLUTE_PATH));
```
Comment (stof) [inline TwigExtension.php]: `addslashes` should not be part of a method named `generateUrl`, as it is not about generating the URL but about escaping it for usage in JS (and so should stay in the function using the URL in a JS context)

Comment (oleg-andreyev) [reply TwigExtension.php]: returned `addslashes` back to `initErrorLogger` and `initLogger`

**Comment by Seldaek** [comment]
@oleg-andreyev I think @stof's point is correct, removing the parameter and just always doing ABSOLUTE_URL would probably make sense. If you can update the PR that'd be great.

**Comment by oleg-andreyev** [comment]
@Seldaek sure. I've reworked my PR to generate an Absolute URL always.

**Pull request merged**